### PR TITLE
Upgrade to use v3.13.1 of OpenLayers

### PR DESCRIPTION
**Pull request by marcjansen**
This PR suggests to update the workshop to use v3.13.1 of OpenLayers. I have also updated some outdated screenshots and added a section with solution to a bonus task.

Please review.

### CODE REVIEW

File: src/layers/proprietary.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Let's add a Bing layer.
   ```js
     source: new ol.source.BingMaps({
       imagerySet: 'Road',
-      key: 'Ak-dzM4wZjSqTlzveKz5u0d4IQ4bRzVI309GxmkgSVr1ewS6iPSrOvOKhA-CJlm3'
+      key: 'AkGbxXx6tDWf1swIhPJyoAVp06H0s0gDTYslNWWHZ6RoPqMpB9ld5FY1WutX8UoF'
```
Comment (ahocevar) [inline src/layers/proprietary.md]: Didn't we want to remove API keys from the workshops? I may be wrong, but I could swear that a pull request from @tschaub got merged a while ago, which removed the API keys.

**Comment by ahocevar** [comment]
Thanks for this @marcjansen. The only comment I have is about Bing API keys. Looks great otherwise.

File: src/layers/proprietary.md (hunk 1 above)
Comment (ahocevar) [inline src/layers/proprietary.md]: #44 is the API key removal pull request. Not merged yet.

**Comment by marcjansen** [comment]
Cheers for the review, @ahocevar. I will now crearte a new release and also update the published version, if you agree.

**Comment by ahocevar** [comment]
Agreed :-)

**Pull request merged**